src/relentless/model/potential/potential.py

```diff
@@ -462,6 +467,10 @@ def derivative(self, type_, var, x):
             deriv = deriv.item()
         return deriv
 
+    def _validate_coordinate(self, x):
+        """Validate the coordinate ``x`` is within the valid range."""
+        pass
+
```

Comment:
I don't think you want to have this here, since it is making an empty implementation of the method (allowing children not to override it).
```suggestion
```